config/_default/menus/menus.en.yaml

```
@@ -2006,6 +2006,11 @@ main:
     parent: setup_pipelines
     identifier: ci_jenkins
     weight: 202
+  - name: Custom spans
+    url: continuous_integration/setup_pipelines/customspans/
```

nit, but wouldn't this be more consistent? 
```suggestion
    url: continuous_integration/setup_pipelines/custom_spans/
```